kubemarine.spec

```diff
@@ -29,7 +29,8 @@ a = Analysis(['./kubemarine/__main__.py'],
                 ('./kubemarine/resources/drop_ins/*',       './kubemarine/resources/drop_ins'),
                 ('./kubemarine/templates/*',                './kubemarine/templates'),
                 ('./kubemarine/templates/plugins/*',        './kubemarine/templates/plugins'),
-                ('./kubemarine/plugins/*',                  './kubemarine/plugins')
+                ('./kubemarine/plugins/*',                  './kubemarine/plugins'),
```

Comment:
This is strange that this directory is in `datas` section but not in `hiddenimports` section. In fact, it is imported dynamically `kubemarine/core/defaults.py`

Reply:
add new procedures